src/10_functions.py

```diff
@@ -2,9 +2,17 @@
 
 # YOUR CODE HERE
 
+def is_even(num):
+    if num % 2 == 0:
+        print('even!')
+    else:
+        print('odd!')
+    
```

Comment:
You might considering have this function return a value, instead of just printing "even" or "odd".  If you do it that way, it's an easy one liner, and then you can use the returned value to determine what to print below.